src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -120,15 +120,7 @@ trait SharedFileSystem {
   def postProcess(backendCall: LocalFileSystemBackendCall): Try[CallOutputs] = {
     implicit val hasher = backendCall.workflowDescriptor.fileHasher
     // Evaluate output expressions, performing conversions from String -> File where required.
-    val outputMappings = backendCall.call.task.outputs map { taskOutput =>
-      val tryConvertedValue =
-        for {
-          expressionValue <- taskOutput.expression.evaluate(backendCall.lookupFunction, backendCall.engineFunctions)
-          convertedValue <- outputAutoConversion(backendCall, taskOutput, expressionValue)
-          pathAdjustedValue <- Success(absolutizeOutputWdlFile(convertedValue, backendCall.callRootPath))
-        } yield pathAdjustedValue
-      taskOutput.name -> tryConvertedValue
-    }
+    val outputMappings = postProcessInner(backendCall, backendCall.call.task.outputs, Seq.empty)
```

Comment:
Hmm there's gotta be a better name than `postProcessInner`... I'll try to come up with some suggestions once I understand everything that function is doing